### PR TITLE
Add contributor experience

### PR DESCRIPTION
**Pull request by jgbarah**
This is based on our discussions during OSS NA

### CODE REVIEW

**Comment by GeorgLink** [comment]
@jgbarah thank you for the pull request.

I have a question and hope that @germonprez  can clarify: Is *Contributor Experience* meant to capture how a contributor experiences the engagment with the community? The quote "asks how satisfied they were with the resolution of the the issue" in #21 reads that way.

I ask because @jgbarah's pull request is about the experience (~knowledge) contributors have accumulated.

I think these two are two different but equally interesting metrics and I want to make sure we have clarity and give them names that are distinguishable.

File: activity-metrics-list.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ We list and describe activity metrics. For different situations, the metrics hav
 |User Groups|user groups perform a variety of crucial marketing, service support, and business-development functions at the grassroots level\\ (Bagozzi & Dholakia, 2006)|
 |[Watchers](activity-metrics/watchers.md)|*Number of watchers* (GitHub)|
 |YouTube Videos|*Number of Youtube videos that mention or specifically deal with the project (e.g. tutorials)*|
+|Contributor Experience|*For each active contributor, time since first contribution*\\ Experienced contributors camm add some unique value to the community, since they carry with them (in part) the history of the project.|
```
[the remark this answers is not in the document]
Comment (jgbarah) [reply activity-metrics-list.md]: Thanks! New commit with the changes. Sorry for not noticing that we were using alphabetic order, and for the typo :-( Yes, your suggestion for the name of the metric is more appropriate.

**Comment by jgbarah** [comment]
> Is Contributor Experience meant to capture how a contributor experiences the engagment with the community? The quote "asks how satisfied they were with the resolution of the the issue" in #21 reads that way.

I agree, those are two different metrics. Maybe we could call the other one "contributor satisfaction", to avoid the many meaning of "experience"?

**Comment by germonprez** [comment]
I'm Ok with this. It may also be that we find that experience is part of one of the other metrics

**Pull request merged**